src/web/routes/application/constants.js

```diff
@@ -1,2 +1,3 @@
 module.exports.YES = 'yes'
 module.exports.NO = 'no'
+module.exports.CHECK_URL = '/check'
```

Comment:
Not to do as part of this PR, but I think the `application` directory needs slightly restructuring; a lot of the files in `application/common` are not actually common and could live in `application/middleware`. This would leave `common` for actual common files such as this `constants.js`:

```
- /application
|- /common
|  |- constants.js
|
| - /middleware
|  |- /render-view
| ...
```